CommonCoreLegacy/build.gradle

```diff
@@ -45,33 +50,34 @@ task(generateiOSStrings, type: org.gradle.api.tasks.JavaExec) {
 }
 
 dependencies {
-  androidTestCompile libs.assertjCore
-  androidTestCompile 'com.android.support.test:runner:0.5'
-  androidTestCompile 'com.android.support.test:rules:0.5'
+  androidTestImplementation libs.assertjCore
+  androidTestImplementation 'com.android.support.test:runner:0.5'
+  androidTestImplementation 'com.android.support.test:rules:0.5'
 
-  testCompile libs.junit
-  testCompile libs.assertjCore
-  testCompile libs.robolectric
-  testCompile 'commons-io:commons-io:2.5'
-  testCompile libs.mockitoKotlin
-  testCompile libs.mockitoCore
-  testCompile libs.jodaTimeTzdb
-  testCompile libs.kluent
+  testImplementation libs.junit
+  testImplementation libs.assertjCore
+  testImplementation libs.robolectric
+  testImplementation 'commons-io:commons-io:2.5'
+  testImplementation libs.mockitoKotlin
+  testImplementation libs.mockitoCore
+  testImplementation libs.jodaTimeTzdb
+  testImplementation libs.kluent
 
   // See https://github.com/skedgo/android-maps-utils.
-  compile 'com.github.skedgo:android-maps-utils:241fc89'
-  compile 'com.android.support:support-annotations:25.3.0'
-  compile libs.playServicesMaps
-  compile libs.jodaTimeAndroid
-  compile libs.rxjava
-  compile 'io.reactivex:rxandroid:1.2.1', { exclude module: 'rxjava' }
-  compile libs.gson
-  compile 'com.github.skedgo:commons-collections:v1.0'
+  implementation 'com.github.skedgo:android-maps-utils:241fc89'
+  implementation 'com.android.support:support-annotations:25.3.0'
+  implementation libs.playServicesMaps
+  implementation libs.jodaTimeAndroid
+  implementation libs.rxjava
+  implementation 'io.reactivex:rxandroid:1.2.1', { exclude module: 'rxjava' }
```

Comment:
Can you please use the value from the dependencies file?

Reply:
yes.